src/it/scala/io/iohk/ethereum/rlp/RLPSpeedSuite.scala

```diff
@@ -56,32 +57,33 @@ class RLPSpeedSuite extends FunSuite
     val ITERATIONS: Int = 10000000
     log.info("Starting " + ITERATIONS + " decoding iterations...")
     val start1: Long = System.currentTimeMillis
-    (1 to ITERATIONS).foreach { _ => RLP.rawDecode(payload); Unit }
+      (1 to ITERATIONS).foreach { _ => RLP.rawDecode(payload); Unit }
```

Comment:
you know, once you refactor the code you're blamed for the code you didn't write